docs/data-sources/column.md

```diff
@@ -2,7 +2,7 @@
 
 The `honeycombio_column` data source retrieves the details of a single column in a dataset.
 
--> **Note** Terraform will fail unless a column is returned by the search. Ensure that your search is specific enough to return a column.
+~> **Warning** Terraform will fail unless a column is returned by the search. Ensure that your search is specific enough to return a column.
```

Comment:
I think I'm still a lil confused what this will end up looking like based on this table. 

<img width="740" alt="image" src="https://github.com/user-attachments/assets/ba6369f9-921d-46ba-80e4-ba54d07bff56">

Will this end up as a yellow callout with the text prefix `**Note** **Warning**`, since it's using `~>`?

I also notice that it says `!>` is supposed to end up with a red callout and the `**Warning**` prefix,  but for all of our callouts that say "Warning" we're using `~>` -- is that what we want? (my guess is yes since they seem like less important warnings, but wanted to check)

Reply:
Providing a `**Something**` overrides the default text prefix, so the above renders like this:

![image](https://github.com/user-attachments/assets/ac1a5477-6352-44e8-b56b-887e0a56deae)

I should have linked their [Doc Preview Tool](https://registry.terraform.io/tools/doc-preview) which lets you try out their flavor of Markdown

Reply:
Got it, thanks! I'll bookmark that for later, I did not realize it was an override but that sounds perfect.